app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,31 @@
 class PostsController < ApplicationController
+  include PostsHelper
+
   def index
     @user = User.find_by_id(params[:user_id])
     @posts = @user.posts
   end
```

Comment:
- [ ] Please the list of posts cannot render so make sure that you fix this error in your posts controller :pray:
  
  ![image](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/92303891/ef3d2906-98ce-4aa8-84a6-ae7cb68f9428)

Reply:
@Owusu-Desmond Hello, can you explain to me please at what moment did you get that error? Because when I started the server (rails s) it shows the following:
![image](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/118059775/bf997ca7-f56e-40cd-b9fc-6d17d370f5d2)

Reply:
Hello, @Owusu-Desmond same as the past comment. can you explain to me please how do you get that error?
![image](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/118059775/4bcabe6e-a72c-4212-bf0f-14bb25b7d5fa)